pandas/_libs/tslibs/dtypes.pyx

```diff
@@ -50,6 +50,25 @@ cdef class PeriodDtypeBase:
         code = offset._period_dtype_code
         return cls(code)
 
+    cpdef int _get_to_timestamp_base(self):
```

Comment:
Can this just be a property? (or no b/c of cpdef)

Reply:
right, no cdef properties until cython3